crates/sui-framework/packages/sui-system/tests/delegation_tests.move

```diff
@@ -488,39 +487,6 @@ module sui_system::stake_tests {
         test_scenario::end(scenario_val);
     }
 
-    #[test]
-    fun test_stake_during_safe_mode() {
```

Comment:
I would like to keep this test or the ability to do safe mode epoch change in Move tests. I'm thinking of hardening the exchange rate lookup during safe mode times. Maybe not for this PR but can we add a native test only function `advance_epoch_safe_mode` that calls the rust version?

Reply:
Agreed. Will follow up after this PR.